src/js/components/AppListItemComponent.jsx

```diff
@@ -105,8 +105,8 @@ var AppListItemComponent = React.createClass({
           {parseFloat(model.totalCpus).toFixed(1)}
         </td>
         <td className="text-right total ram">
-          <span title={`${model.totalMem}MB`}>
-            {`${ViewHelper.convertMegabytesToString(model.totalMem)}`}
+          <span title={`${model.totalMem} MiB`}>
+            {`${Util.filesize(model.totalMem * Math.pow(1024, 2), 0)}`}
```

Comment:
True, the naming isn't the best anymore, before it was in Units.filesize(), but now in Utils it isn't descriptive enough anymore. Could be named ```convertToReadableFilesize``` or something like that.
Otherwise look what @orlandohohmeier writes just now.. ;)